unlock-app/package.json

```diff
@@ -36,6 +36,7 @@
     "babel-loader": "8.0.6",
     "babel-plugin-require-context-hook": "1.0.0",
     "babel-plugin-styled-components": "1.10.6",
+    "babel-preset-react-app": "^9.1.0",
```

Comment:
Storybook failed to start without this. Not certain why, but it makes sense to add it here.